doc/radosgw/lua-scripting.rst

```diff
@@ -297,6 +302,23 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Background Context
+--------------------
+The background context may be used for various need such as analytics, monitoring and
+caching data from other context executions.
+
```

Comment:
The \`\`background\`\` context may be used for purposes that include analytics, monitoring, caching data for other context executions.